### DISCUSSION TITLE
Welcome to helpdesk/discussions!

### DISCUSSION THREAD

**Discussion by suzialeksander** [geneontology/helpdesk#340] [General]
## 👋 Welcome!
  We’re GO-ing to use Discussions as a place to connect with GO-pros, the GO-curious, and other all other members of the Gene Ontology community. We hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we build together 💪

**Comment by SangiSa** [comment]
Hi, I am new to RNA seq. analysis. I have glMDPlot which provides HTML file. I tried to use the following code for species Md instead of Mm. Md is not available is database, so I got error! Can you please help me with a possible way to proceed in such a case? 
go <- goana(fit.cont, coef="B.PregVsLac",species = "Mm")
topGO(go, n=10)

**Comment by chorong5155** [comment]
Hi 
I just have a general question as to how we can register new GO term annotations.,.
so if I want to analyze GO term on certain viruses only, what would be the first step?
Would this be something hard to do?
